pom.xml

```diff
@@ -297,7 +298,7 @@
 									<groupId>org.eclipse.emfcloud.modelserver</groupId>
 									<artifactId>org.eclipse.emfcloud.modelserver.parent</artifactId>
 									<classifier>releng/org.eclipse.emfcloud.modelserver.releng.target/targetplatform</classifier>
-									<version>${project.version}</version>
+									<version>0.7.0-SNAPSHOT</version>
```

Comment:
why not use the project version anymore?

Reply:
Because of the introduction of external libraries, which are not versioned as "0.7.0", the target platform couldn't resolve. The alternative would be to override the target platform definition for these bundles that use a different version, but I think having a single target platform is more consistent (rather than assuming we'll have a target platform version per bundle version).

Reply:
Just as a small sidetrack here: For us it is in general easier if we stick to the variable when we want to publish a release (candidate) as we can then use our scripts to adapt the versions without having to check manually.

Reply:
If the target platform itself is not published, you can achieve the same result by using a fixed version (There's no need to ever change the target platform version; you can set it to `1.0.0` or `0.0.1` and forget about it).

Alternatively, you can use an explicit version Property (instead of the predefined `${project.version}`), to keep the Target Platform versioned. This way, the Target Platform can still inherit the version from the parent, but projects referencing the target platform may be versioned separately (while still using this shared target platform at a specific version).

The idea is really that the target platform should have its own version, rather than reuse the version from the project referencing it.